testing/asserts.ts

```diff
@@ -1,15 +1,16 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-import { assertEquals as prettyAssertEqual } from "./pretty.ts";
+import { createStr, buildDiffMessage, buildMessage } from "./pretty.ts";
+import diff, { DiffResult } from "./diff.ts";
 
 interface Constructor {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   new (...args: any[]): any;
 }
 
 export class AssertionError extends Error {
-  constructor(message: string) {
+  constructor(message?: string) {
     super(message);
-    this.name = "AssertionError";
+    this.name = "Assertion Error";
```

Comment:
Why? I think the name of an error should be the same as its class name?

Reply:
I agree...  It should be `AssertionError`

```
> new RangeError().name
"RangeError"
> new TypeError().name
"TypeError"
```

Reply:
Seems legit.